lib/ansible/modules/cloud/kubevirt/kubevirt_pvc.py

```diff
@@ -126,6 +126,21 @@
       - "This uses the DataVolume source syntax:
         U(https://github.com/kubevirt/containerized-data-importer/blob/master/doc/datavolumes.md#https3registry-source)"
     type: dict
+  wait:
+    description:
+      - "If set, this module will wait for the PVC to become bound and CDI (if enabled) to finish its operation
+        before returning."
+      - "Used only if I(state) set to C(present)."
+      - "Unless used in conjuction with I(cdi_source), this might result in a timeout, as clusters may be configured
```

Comment:
Maybe we can add this on TODO to detect it.

Reply:
Actually, this might be doable in a few lines…

Reply:
Nope. Maybe some time in the future.